src/editors/sceditor.js

```diff
@@ -13,41 +13,36 @@ export var ScEditor = StringEditor.extend({
     this.input.setAttribute('data-schemaformat', this.input_type)
   },
   afterInputReady: function () {
-    var self = this; var options
-
-    if (window.jQuery && window.jQuery.fn && window.jQuery.fn.sceditor) {
+    if (window.sceditor) {
       // Get options, either global options from "this.defaults.options.sceditor" or
       // single property options from schema "options.sceditor"
-      options = this.expandCallbacks('sceditor', $extend({}, {
-        plugins: self.input_type,
+      var options = this.expandCallbacks('sceditor', $extend({}, {
+        format: this.input_type === 'html' ? 'xhtml' : 'bbcode',
```

Comment:
Just noticed this, which might be a problem. Since the `html` type is already used by the Ace Editor (See defaults.js).
So it should only work if type is set explicit to `xhtml` or `bbcode`. (Again, see defaults.js)